cdragontoolbox/tftdata.py

```diff
@@ -180,6 +180,7 @@ def parse_items(self, map22):
                 "name": item.getv(0xC3143D66),
                 "desc": item.getv(0x765F18DA),
                 "icon": item.getv("mIconPath"),
+                "unique": item.getv(0x9596A387, false),
```

Comment:
It's `False`, not `false`. ;)

Reply:
Why does python have to be the odd child xD

Reply:
Fixed with ddebdd5d33196999eba0bb0c6715ecd1596af98c